fog/ingest/enclave/impl/src/lib.rs

```diff
@@ -143,13 +143,12 @@ impl<OSC: ORAMStorageCreator<StorageDataSize, StorageMetaSize>> SgxIngestEnclave
 
                 // Create a TxOutRecord, flattening the Txo data and getting extra data like
                 // global index, block index, timestamp.
-                let fog_tx_out = FogTxOut::from(txo);
                 let meta = FogTxOutMetadata {
                     global_index: chunk.global_txo_index + index as u64,
                     block_index: chunk.block_index,
                     timestamp: chunk.timestamp,
                 };
-                let txo_record = TxOutRecord::new(fog_tx_out, meta);
+                let txo_record = TxOutRecord::new(fog_tx_out.clone(), meta);
```

Comment:
There is some discussion in the README of mc-oblivious repo

At this particular line, we're not worried about constant-time, because we aren't trying to be constant-time with respect to which fog_tx_out we are processing, the operator already knows / can infer that. We're just trying to be constant time with respect to the contents of the decrypted fog hint